libs/wingsdk/src/std/array.ts

```diff
@@ -125,6 +145,19 @@ export class Array {
     searchElement;
     throw new Error("Macro");
   }
+
+  /**
+   * Returns a sorted copy of the array.
+   *
+   * @macro ($self$.sort($args$ ?? undefined))
```

Comment:
```suggestion
   * @macro ($self$.sort($args$))
```

I checked the compiler code and this undefined actually shouldn't be necessary (my bad!)